transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -205,7 +205,7 @@ public void transform(StructuredRecord in, Emitter<StructuredRecord> emitter) th
         }
       }
     } catch (IOException e) {
-      throw Throwables.propagate(e);
+      emitter.emitError(new InvalidEntry<>(31, e.getStackTrace()[0].toString() + " : " + e.getMessage(), in));
```

Comment:
This is  not something which is standard for CSV. I referred one of the exiting implementation of emitError in XMLParser. Could you please suggest the error code format ?